images/airflow/2.9.2/python/mwaa/celery/task_monitor.py

```diff
@@ -70,6 +72,14 @@
 # consumption, we reset the worker to non-idle state and backoff from checking for
 # idleness for a minute.
 IDLENESS_RESET_BACKOFF_PERIOD = timedelta(minutes=1)
+# In case of issues, signals can arrive late and out of order. So, we scan all unprocessed signals from the last one hour.
```

Comment:
Can you make all the comments generic related to the timedelta?  They have a tendency to be missed when someone adjusts the duration and then it will be confusing.

Reply:
Not sure if I follow.